docs/use-hotkeys.md

```diff
@@ -0,0 +1,47 @@
+# useHotkeys
+
+Adds one new behavior to your Stimulus controller : `hotkeysmemo`
+
+This behavior can be used to register hotkeys using the [hotkeys-js](https://wangchujiang.com/hotkeys/) library. To use it, do the following:
+
+1. Install the hotkeys-js peer dependency:
+
+```bash
+$ yarn add hotkeys-js
+```
+
+2. Define hotkeys and respective handlers in a static `hotkeys` object.
+
+## Reference
+
+```javascript
+bindHotkeys(controller)
+unbindHotkeys(controller)
+```
+
+**controller** : a Stimulus Controller (usually `'this'`)
+
+## Usage
+
+**Composing**
+
+```js
+import { Controller } from 'stimulus'
+import { bindHotkeys, unbindHotkeys } from 'stimulus-use'
+
+export default class extends Controller {
+  static hotkeys = {'cmd+a': 'handler'}
```

Comment:
We somewhat agreed that we shouldn't introduce more `static` configurations.

It would also makes sense here. Instead you would pass the options directly with the `bindHotkeys` function.

Some like: 

```js
bindHotkeys(this, {
  hotkeys: {
    'cmd+a': 'handler'
  }
}
```

This would also allow for other options to be specified.

Reply:
Ah, didn't know that. If that is the general contract, I'll happily refactor.